website/docs/getting-started/setup/instance-configuration/google-maps.mdx

```diff
@@ -0,0 +1,69 @@
+---
+description: Follow these steps to configure Google Maps Key for your self-hosted Appsmith instance.
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Google Maps
+
+This page provides instructions to integrate Google Maps to use the [Map Widget](/reference/widgets/maps) in your applications on the self-hosted instance.
+
+## Prerequisites
+
+- A [Google Cloud account](https://www.google.com/aclk?sa=l&ai=DChcSEwi0373S7OX_AhW7Vn0KHT5BD30YABABGgJzZg&sig=AOD64_3YNMGHJAu3vGJmjK-vrl8qni2BxA&q&adurl&ved=2ahUKEwjjs7XS7OX_AhWy3TgGHWtHAkYQ0Qx6BAgIEAE). 
+
+## Setup Google Maps platform
```

Comment:
```suggestion
## Generate and restrict API key
```

Reply:
Why we want to specify this heading. Basically, with generation and restriction we are setting up a Google Maps platform.

Reply:
Because we are technically not setting up any platform. We are just generating API key. Language-wise it seems to be better to specify what exactly they will be doing. 

Even just **Generate API key** will do. Restrict is a secondary step

Reply:
Actually generating and restricting is a part of setup only on Google Maps Platform. Nevertheless, **Generate Google Maps API key** looks okay to me. So, adding that as heading.